components/Form/AddFlightForm.js

```diff
@@ -21,12 +21,12 @@ export default function AddFlightForm() {
 
   const router = useRouter();
 
-  const [isFocused, setIsFocused] = useState({});
+  const [$isFocused, set$isFocused] = useState({});
```

Comment:
The state variable doesn't need the dollar sign. Only the prop on the styled components needs it.